com/win32comext/mapi/src/PyIMsgStore.i

```diff
@@ -79,6 +79,32 @@ PyObject *PyIMsgStore::OpenEntry(PyObject *self, PyObject *args)
 
 %}
 
+%native(StoreLogoff) StoreLogoff;
+%{
+// @pyswig <o PyIInterface>|StoreLogoff|Enables the orderly logoff of the message store.
+PyObject *PyIMsgStore::StoreLogoff(PyObject *self, PyObject *args) 
+{
+    HRESULT  _result;
+    unsigned long  flags;
+
+	IMsgStore *_swig_self;
+	if ((_swig_self=GetI(self))==NULL) return NULL;
+	// @pyparm int|flags||Bitmask of flags that controls how the message store is closed.
+    if(!PyArg_ParseTuple(args,"l:StoreLogoff", &flags)) 
```

Comment:
I'm quite sure this isn't consistently done, but new code should probably use the format flag `k`, documented at "k ([int](https://docs.python.org/3/library/functions.html#int)) [unsigned long]

    Convert a Python integer to a C unsigned long without overflow checking."

Reply:
Thanks Mark, I've addressed both comments